PlantFriends.ino

```diff
@@ -124,23 +132,37 @@ static volatile uint16 publishedPacketId = -1;
 // BME280 sensor
 Adafruit_BME280 bme280; // Connect to the BME280 via I2C
 
-/*
-long getMedian(uint32_t *array, size_t n){
-  //n = sizeof(array[0])/sizeof(array);
 
-  // Sort array
-  for (size_t i = 0; i < n; i++){
-    for (size_t j = 0; j < n; j++){
-      if (array[j] > array[i]){
-        uint32_t tmp = array[i];
-        array[i] = array[j];
-        array[j] = tmp;
+static void bubbleSort(uint32_t *array, size_t n) {
+  //size_t n = sizeof(array[0])/sizeof(array);
+  size_t new_n;
+  //unsigned long n=len;
+  uint32_t temp;
+   while(n > 1){
+    new_n=0;
+    for(size_t i = 1; i < n ; i++){
+      if(array[i-1]>array[i]){
+        temp=array[i];           //swap places in array
+        array[i]=array[i-1];
+        array[i-1]=temp;
+        new_n=i;
       }
+    }
+    n=new_n;
```

Comment:
Added space before and after equal sign i.e. `n = new_n;`. The same for the entire function.